src/jit/ee_il_dll.cpp

```diff
@@ -163,6 +163,20 @@ CorJitResult CILJit::compileMethod (
         return g_realJitCompiler->compileMethod(compHnd, methodInfo, flags, entryAddress, nativeSizeOfCode);
     }
 
+    if (!JitConfig.isInitialized())
```

Comment:
I'm guessing some obfuscated apps are not going to work. Unless the obfuscator passes a "real" VM ICorJitInfo* and not one they make themselves. Maybe that's already true.